www/NativeRingtones.js

```diff
@@ -42,6 +46,9 @@ RingtoneManager.prototype.playRingtone = function (ringtoneUri, playOnce, volume
         streamType = -1;
     }
 
+    if(typeof completeCallback === "function") {
```

Comment:
Inspirado de: https://github.com/we-are-Joinup/cordova-plugin-nativeaudio/blob/master/www/nativeaudio.js#L41